common/common.go

```diff
@@ -457,13 +457,13 @@ func stringToSlice(in string) string {
 	return buffer.String()
 }
 
-// ExtractEnvConfig extract env vars from configuration.
-func ExtractEnvConfig(configs []string) map[string]string {
+// ExtractConfig parses key-value configuration into a map
+func ExtractConfig(configs []string) map[string]string {
 	c := make(map[string]string)
 	for _, v := range configs {
 		kv := strings.SplitN(v, "=", 2)
 		if len(kv) == 2 {
-			c[kv[0]] = os.ExpandEnv(kv[1])
+			c[kv[0]] =kv[1]
```

Comment:
add fmt on save to your editor already ;)

Reply:
sorry - had it fn , just not CLI